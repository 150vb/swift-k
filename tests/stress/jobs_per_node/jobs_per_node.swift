type file;
type script;

app (file out, file err) app_run (script run)
{
    bash @run stdout=@out stderr=@err;
}

file f_out[] <simple_mapper; prefix="output/t", suffix=".out">;
file f_err[] <simple_mapper; prefix="output/t", suffix=".err">;

script wrapper <"count_jobs.sh">;

int loop = @toInt(@arg("loops","10"));

foreach item,i in [0:loop-1] {
	(f_out[i], f_err[i]) = app_run(wrapper);
}
