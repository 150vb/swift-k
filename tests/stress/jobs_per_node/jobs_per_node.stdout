Swift 0.94 swift-r6888 cog-r3762

RunID: 20130822-2202-c5ih1pj1
